Core/DISET/private/Transports/BaseTransport.py

```diff
@@ -36,7 +38,7 @@ def __init__( self, stServerAddress, bServerMode = False, **kwargs ):
     self.sentKeepAlives = 0
     self.waitingForKeepAlivePong = False
     self.__keepAliveLapse = 0
-    self.oSocket = None
+    self.oSocket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
Why do you need this ? The self.oSocket will anyway be recreated in initAsXXX() functions